src/aktualizr_lite/CMakeLists.txt

```diff
@@ -0,0 +1,30 @@
+if(BUILD_OSTREE)
+add_executable(aktualizr-lite main.cc)
+target_link_libraries(aktualizr-lite aktualizr_static_lib ${AKTUALIZR_EXTERNAL_LIBS})
+
+install(TARGETS aktualizr-lite RUNTIME DESTINATION bin COMPONENT aktualizr-lite)
+
+set(TEST_SOURCES test_list.sh)
+
+if((CMAKE_BUILD_TYPE MATCHES "Valgrind") AND (NOT AKTUALIZR_TEST_NO_VALGRIND))
```

Comment:
The `NO_VALGRIND` is an argument that can be given to `add_aktualizr_test`, for a given test.

It doesn't apply in this case, the condition should just be `if(CMAKE_BUILD_TYPE MATCHES "Valgrind")`